src/app.js

```diff
@@ -0,0 +1,15 @@
+import vomitUnicorn from './vomit-unicorn.js';
+//selecting elements I want to use, (posting the cast list)
+const colorButtons = document.querySelectorAll('.color');
+const unicorn = document.getElementById('unicorn');
+
+//initializing stuff and creating my loop for my buttons
+for(let i = 0; i < colorButtons.length; i++) {
+    const colorButton = colorButtons[i];
+
+    //wiring up listening event
+    colorButton.addEventListener('click', () => {
+        vomitUnicorn(unicorn, colorButton.value);
+        console.log('in event listener');
```

Comment:
Be sure to delete `console.logs` before merging with production code! :-)

Reply:
Thank you, it has since been removed, and now there is A BUNCH MORE COOL STUFF!